components/settings/SettingChooser.vue

```diff
@@ -40,6 +41,8 @@ export default class SettingChooser extends Vue {
   }
 
   set selected(value) {
+    const { getInstance: Api } = Connector
+    Api().connect(value)
```

Comment:
Seems Connector could connect value only when`type === 'setApiUrl'`.  👀 
ref: src/store/index.ts. line 57

Reply:
You are right 👍.
Already fix it.